Updates to support new format of queries that offer pagination

briri opened the pull request:
## Description

Fixes [#180](https://github.com/CDLUC3/dmsp_backend_prototype/issues/180)

Requires backend PR [#267](https://github.com/CDLUC3/dmsp_backend_prototype/pull/267)

- Update graphQL queries that now offer pagination. 

This PR only updated enough to keep current functionality working. We will want to create additional tickets to go back and update the existing queries to use pagination fully

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

- Linter, and Jest tests were updated and passing
- Manually verified in the UI

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I updated the CHANGELOG.md and added documentation if necessary
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have completed manual or automated accessibility testing for my changes
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

jupiter007 commented on app/[locale]/template/[templateId]/page.tsx:
```diff
@@ -402,7 +401,9 @@ const TemplateEditPage: React.FC = () => {
-      <Modal isDismissable
+      <Modal
+        isDismissable
+        onOpenChange={setPublishModalOpen}
```
I'm seeing changes in your PR that are already in `development` branch. Like L406 "onOpenChange" is already there. Did you work off of the latest development branch?

jupiter007 replied:
Oh It's because your PR is merging into `main` and not `development` branch

jupiter007 commented on lib/graphql/client.ts:
```diff
@@ -53,5 +53,4 @@ export const { getClient } = registerApolloClient(() => {
     });
-});
-
+});
```
? Why is this change here?

jupiter007 commented on lib/graphql/client/apollo-client.ts:
```diff
@@ -16,9 +16,4 @@ export const createApolloClient = () => {
-
-
-
+}
```
Why is this change here?

jupiter007 commented on lib/graphql/queries/affiliations.ts:
```diff
@@ -3,7 +3,11 @@ import { gql } from '@apollo/client';
   affiliations(name: $name) {
-    id
-    name
+    totalCount
```
You might want to check the pages where these queries are used. For example, it looks like the TypeAhead search in the `/account/profile` page is no longer working.

briri commented:
Thanks for the review @jupiter007 

I updated the profile page to use the `items` array  and updated to remove some changes made by my editor (I guess 🤷🏻‍♂️) that removed some blank lines at the end of a few files.

jupiter007 reviewed (approved):
Thanks for making the change Brian.
